moveit_experimental/CMakeLists.txt

```diff
@@ -56,11 +57,10 @@ catkin_package(
     pluginlib
   DEPENDS
     Boost
-    Eigen3
```

Comment:
The include directories are exported already, so is there any need to declare Eigen3 as a dependency, considering Eigen is completely header only?